homeassistant/components/ollama_conversation/strings.json

```diff
@@ -0,0 +1,32 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "url": "[%key:common::config_flow::data::url%]",
+          "model": "Model name",
+          "prompt": "Prompt template",
+          "model_options": "Model options",
+          "max_history": "Max history messages"
```

Comment:
These should all be removed from initial user setup.
```suggestion
```

Reply:
I can't make the initial call without a model though.